docs/solutions/sql_server_always_on_availability_groups/README.md

```diff
@@ -1 +1,35 @@
-## Coming Soon!
+Solution: MS SQL Always On.
+
+This Terraform template implements the infrastructure needed to fully deploy a cross AD, MS SQL Always On cluster.  The cluster can be spread across either 2 or 3 ADs, depending on your needs/preferences, but cannot be located in a single AD.
```

Comment:
"...deploy a cross AD, MS SQL Always On cluster."
Should be:
"...deploy an MS SQL Always On cluster across ADs."
or 
"...deploy an MS SQL Always On cluster across availability domains."

Reply:
Done